moneo.py

```diff
@@ -197,6 +195,10 @@ def deploy_work_docker(self, hosts_file, max_threads=16):
         logging.info('Deploying docker container to workers')
         out = pscp(copy_path, destination_dir, hosts_file, user=self.args.user)
         logging.info(out)
+        copy_path = './moneo_config.json'
+        destination_dir = '/tmp/moneo-worker'
```

Comment:
Are these always the same? If so, definite them at the top rather than repeating them?

Reply:
I will address

Reply:
addressed